WikiTemplate/Crypto/wikitemplate-cryptoDesktop.md

```diff
@@ -0,0 +1,150 @@
+
+## Onboarding
+
+- [ ] Verify loading `brave://wallet` on a clean profile shows onboarding flow
+- [ ] Verify you are able to complete the onboarding flow to Ethereum create wallet by default
+- [ ] Verify you are able to complete the backup seed phrase during onboarding itself
+- [ ] Verify you can skip the onboarding from backup wallet screen and still create a wallet
+- [ ] Verify when a Dapp interacts without creating wallet opens a new tab and shows onboarding
+
+## Importing Wallet
+
+- [ ] Verify with MetaMask extension installed and wallet created, visiting `brave://wallet/crypto/onboarding/import-or-restore` shows option to import account from MetaMask during onboarding
+- [ ] Verify error message is shown when trying to use same password as MetaMask and it doesn't meet Brave Wallet's password criteria
+- [ ] Verify able to successfully import MetaMask wallet into Brave Wallet
+- [ ] Verify with Crypto Wallet Extension installed and wallet created, visiting `brave://wallet/crypto/onboarding/import-or-restore` shows option to import account from old Crypto Wallet Extension
+- [ ] Verify error message is shown when trying to use same password as Crypto Wallet Extension and it doesn't meet Brave Wallet's password criteria
+- [ ] Verify able to successfully import Crypto Wallet Extension wallet into Brave Wallet
```

Comment:
Verify that legacy Crypto wallets with 12 & 24 word recovery phrases can be imported.

Reply:
Will separate it to two tests